src/options/smt_options.toml

```diff
@@ -207,12 +207,21 @@ header = "options/smt_options.h"
   type       = "bool"
   help       = "after UNSAT/VALID, produce and check an unsat core (expensive)"
 
+# Temporary option until old unsat cores are removed and this becomes the standard
+[[option]]
+  name       = "unsatCoresNew"
+  category   = "regular"
+  long       = "produce-unsat-cores-new"
+  type       = "bool"
+  help       = "turn on unsat core generation using the new proof infrastructure. Note this option will activate the --produce-proofs option and that it is incompatible with the regular unsat cores."
```

Comment:
I would omit saying this activates the `--produce-proofs` option since it (should not) be equivalent to enabling it on the command line.  This is just an internal detail.